partner_contact_job_position/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2014 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2015 Antonio Espinosa <antonioea@antiun.com>
+# Copyright 2015 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    job_position_id = fields.Many2one(
+        "res.partner.job_position",
+        "Categorized job position",)
```

Comment:
If you put the closing ')' on the last line of the field definition (a style I definitely like, it goes well with the python philosophy on indenting), you are supposed not to have the trailing comma, and certainly not without following it by a space. So best to just leave the comma out.